src/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSourceActivity.cs

```diff
@@ -35,19 +35,19 @@ public string Id
         }
         public System.Diagnostics.ActivityIdFormat IdFormat { get { throw null; } }
         public string OperationName { get { throw null; } }
-        public System.Diagnostics.Activity Parent { get { throw null; } }
-        public string ParentId { get { throw null; } }
+        public System.Diagnostics.Activity? Parent { get { throw null; } }
+        public string? ParentId { get { throw null; } }
         public System.Diagnostics.ActivitySpanId ParentSpanId { get { throw null; } }
         public bool Recorded { get { throw null; } }
-        public string RootId { get { throw null; } }
+        public string? RootId { get { throw null; } }
         public System.Diagnostics.ActivitySpanId SpanId { get { throw null; } }
         public System.DateTime StartTimeUtc { get { throw null; } }
         public System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, string>> Tags { get { throw null; } }
         public System.Diagnostics.ActivityTraceId TraceId { get { throw null; } }
-        public string TraceStateString { get { throw null; } set { } }
+        public string? TraceStateString { get { throw null; } set { } }
         public System.Diagnostics.Activity AddBaggage(string key, string value) { throw null; }
         public System.Diagnostics.Activity AddTag(string key, string value) { throw null; }
```

Comment:
Should baggage and tag values both allow null?

Reply:
yes, was not updated ref assemblies yet